### ISSUE TITLE
bug: init app throws .env file not found error

### ISSUE THREAD

**Issue by Emptyless** [liamwh/Veloxide#111]
### Did you check docs and existing issues?

- [X] I have read all the Veloxide docs
- [X] I have searched the existing issues of Veloxide

### Database kind

n/a

### Operating System

MacOS 13.5.1

### Describe the bug

While initializing, the app throws:

```
Copying .env.example to .env
FAIL - No such file or directory (os error 2)
```

### Steps To Reproduce

1. `brew install rust just protobuf`
2. `cargo install veloxide`
3. `veloxide init veloxide-demo`

### Expected Behavior

The app to initialize

**Comment by liamwh** [comment]
Thanks for highlighting this. Since I recently shuffled the directory structure of the repo around this has broken the CLI tool as the path to copy the .env.example file was hardcoded. Will get onto fixing this soon. Thanks again 🙏

**Comment by liamwh** [comment]
Closed with [v2.0](https://github.com/liamwh/Veloxide-CLI/releases/tag/v2.0.0) of the [Veloxide CLI](https://github.com/liamwh/Veloxide-CLI)
